### PR TITLE
Add support for ILC'd AppX tests.

### PR DESCRIPTION
**Pull request by crummel**
- Add native mode support to xunit.console.uwp.
- Update targets to fix conflicts between AppX and ILC'd tests.
- Some cleanup in logging for AppX runner.

### CODE REVIEW

File: src/xunit.console.uwp/HostedAppxTest.cs [hunk 1]
```diff
@@ -132,27 +140,28 @@ public void Run(bool debug, TimeSpan timeout)
...
-                Console.WriteLine("Killing {0}", pid);
-                p.Kill();
+                var p = Process.GetProcessById(pid.ToInt32());
+                Console.WriteLine("Running {0} in process {1} at {2}", p.ProcessName, pid, DateTimeOffset.Now);
+                while (timer.ElapsedMilliseconds < timeout.TotalMilliseconds && !p.HasExited)
+                {
+                    Thread.Sleep(1000);
+                }
+                var cleanExit = p.HasExited;
+                if (!p.HasExited)
```
Comment (MattGal) [inline src/xunit.console.uwp/HostedAppxTest.cs]: Should use cleanExit here.  It's possible for the property's value to change between these two lines resulting in saying "I'm killing this, and it cleanly exited!"

**Comment by MattGal** [comment]
LGTM.  Consider using interpolated strings in the future, they make reading strings much easier.

File: src/xunit.console.uwp/HostedAppxTest.cs (hunk 1 above)
Comment (crummel) [inline src/xunit.console.uwp/HostedAppxTest.cs]: Good catch, fixed.

**Comment by crummel** [comment]
Build finished.

**Comment by Chrisboh** [comment]
No action in any recent amount of time.  Please reopen if needed.

**Pull request closed without merging**